tests/ReactTestApp/android/app/build.gradle

```diff
@@ -120,16 +120,16 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  */
 def enableHermes = project.ext.react.get("enableHermes", false);
 
+/**
+ * Architectures to build native code for in debug.
+ */
+def nativeArchitectures = project.getProperties().get("reactNativeDebugArchitectures")
+
 android {
     ndkVersion rootProject.ext.ndkVersion
 
     compileSdkVersion rootProject.ext.compileSdkVersion
 
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
Have these been removed to ensure that the SDK's "native" Java version is used?
Java is pretty volatile, so I just want to make sure that this is a calculated move.

Reply:
I'm going to remove this entire app.